### ISSUE TITLE
Accessibility: When trying to move or copy a file, I can no longer pick the destination folder

### ISSUE THREAD

**Issue by the-swissionary** [nextcloud/server#16494]
Hello,
Just encountered this annoying bug using NextCloud Server 16.0.3. I tried it using an iPhone 7 running IOS Safari as well as on a Windows 10 machine running Microsoft Edge and Firefox. I am running Assistive Technologies: VoiceOver on IOS and NVDA and Narrator on Windows 10 - I tried with all of them.

Issue:
When trying to copy or move a file to a new location, the user cannot select the Destination folder when Assistive Technology is enabled.
My guess is that the Destination Folder Dialogue is not usable for keyboard users either.

To Reproduce:
1) Open a NextCloud Instance in a browser.
2) Select one or more files, then click the "Actions" button; NOTE: On IoS, I could not even complete this step reliably. Clicking this ACTIONS Button always resultet in NextCloud actually opening the file...
3) Select "Copy or Move". The Destination Folder Dialogue should appear.
4) If you haven't done so, fire up the Screen Reader of your choice (Windows Narrator and IOS VoiceOver are both Built-In...) Now try to pick a folder from the list. Do try doing this without a mouse. By either using the arrow keys or the TAB Key.
5) Pick either "Copy" or "Move".

Expected Behavior: One should be able to select a folder, to which the previously selected file is then copied or moved.

Actual Behavior: As one can't really select a folder, NextCloud attempts to copy or move the file into the folder in which it is already placed. This results in having either two copies of the same file in one location; or NextCloud throwing an error, saying it can't move the file as it already existed in that location.

Please do fix this soon. We desperately need an accessible NextCloud interface here. We do rely on it daily and it is simply frustrating if such basic tasks as moving or copying files can't be performed without the help of sighted people.

Sadly, the NextCloud for IOS App is not making any progress whatsoever towards becoming more accessible either. Thus, we can't even use this one to copy or move files.

**Comment by kesselb** [comment]
cc @nextcloud/accessibility

**Comment by tyrylu** [comment]
Confirmed, respectively i am able to copy/move to the first folder in the tree. Thre rest can not be clicked on.

**Comment by aignerat** [comment]
hey the-swissonary,

ever tried the clients or webdav? Could be a suitable quick solution for you.

regards,

chris

**Comment by the-swissionary** [comment]
Hello,
> ever tried the clients or webdav? Could be a suitable quick solution for you.
Yes, I am currently using WebDAV and the client software; they are the only way for me to move files. Sadly, the IOS Client has some accessibility issues as well. So I really do hope this issue gets addressed soon.

**Comment by skjnldsv** [comment]
cc @jancborchardt

**Comment by jancborchardt** [comment]
I’m so sorry for the lateness @the-swissionary! I can totally reproduce this, also with using the keyboard on desktop.

The first part of the fix (making the entries tabbable for keyboard and screenreader) is at https://github.com/nextcloud/server/pull/19807. For the second part, so that pressing Enter actually opens and goes into a folder, I need Javascript help e.g. from @skjnldsv or @juliushaertl. :)

**Comment by the-swissionary** [comment]
Thankkk you @jancborchardt and everybody for looking into this. I am glad the discussion is moving again. Looking forward to a more accessible NextCloud experience! :)

**Comment by szaimen** [comment]
Hi, please update to 24.0.8 or better 25.0.2 and report back if it fixes the issue. Thank you!
